flows/codelingo/review/review/cli.go

```diff
@@ -166,16 +169,67 @@ func reviewAction(cliCtx *cli.Context) (chan proto.Message, <-chan *flowutil.Use
 		return nil, nil, nil, nil, errors.Trace(err)
 
 	}
+	// Build `path` for go projects as "reponame" for a local repo or "github.com/username/reponame" for a github repo
```

Comment:
```suggestion
	// Path is the relative path from $GOPATH/src to the repo root. It's required for package resolution.
```

There's no need to mention the value of path for a github repo here, since this is the CLI and it only concerns local repos.

Reply:
Please put path generation into its own function; the `reviewAction` function is about 200 lines long!